ghost/core/test/unit/server/services/email-service/email-renderer.test.js

```diff
@@ -2593,6 +2599,10 @@ describe('Email renderer', function () {
             await testButtonBorderRadius('pill', '9999px');
         });
 
+        it('sets imageCorners to correct rounded value (emailCustomizationAlpha)', async function () {
+            await testImageCorners('rounded', 'border-radius: 6px;');
+        });
+
```

Comment:
Would be good to test all potential values - we're missing tests that verify `'square'` or an unknown/missing value doesn't also give us rounded corners.